bgpd/bgp_rd.c

```diff
@@ -132,6 +129,7 @@ int str2prefix_rd(const char *str, struct prefix_rd *prd)
 		if (!all_digit(half))
 			goto out;
 
+		s = stream_new(8);
```

Comment:
Can we reuse RD_BYTES instead of numeric?

Reply:
Thanks for review, done.